doc/api/crypto.md

```diff
@@ -1032,8 +1037,18 @@ added: v0.1.92
 -->
 
 Verifies the provided data using the given `object` and `signature`.
-The `object` argument is a string containing a PEM encoded object, which can be
-one an RSA public key, a DSA public key, or an X.509 certificate.
+The `object` argument can be either a string containing a PEM encoded object,
+which can be one an RSA public key, a DSA public key, or an X.509 certificate,
+or an object with some of the following properties:
+
+* `key`: {string} - PEM encoded private key (required)
+* `padding`: {string} - RSA padding, either `'pkcs1'` for RSASSA-PKCS1-v1_5
+  (default) or `'pss'` for RSASSA-PSS
+* `saltLength`: {number} - salt length for RSASSA-PSS. If this is set to `-1`,
+  the salt length will be set to the digest size. A value of `-2` (default)
```

Comment:
Can you use the constant name instead of `-2` for specifying the default? Ideally, users won’t have to care what particular value the constant has